peps/pep-0788.rst

```diff
@@ -0,0 +1,603 @@
+PEP: 788
+Title: Reimagining native threads
+Author: Peter Bierma <zintensitydev@gmail.com>
+Sponsor: Victor Stinner <vstinner@python.org>
```

Comment:
cc @vstinner for confirmation

Reply:
Yes, I'm discussing with Peter, I agreed to sponsor the PEP.